app/screens/send-bitcoin-screen/send-bitcoin-details-screen.tsx

```diff
@@ -385,15 +413,7 @@ const SendBitcoinDetailsScreen: React.FC<Props> = ({ route }) => {
             <TouchableWithoutFeedback
               key={wallet.id}
               onPress={() => {
-                setPaymentDetail(
-                  (paymentDetail) =>
-                    paymentDetail &&
-                    paymentDetail.setSendingWalletDescriptor({
-                      id: wallet.id,
-                      currency: wallet.walletCurrency,
-                    }),
-                )
-                toggleModal()
+                chooseWallet(wallet as unknown as Wallet)
```

Comment:
this type of casting is not good. you should find the root cause and fix the type instead of remove type safeguard.